### PR TITLE
Add email & country

### PR DESCRIPTION
**Pull request by soumyadip007**
Fixes #

#### Describe the changes you have made in this pr - I have added Email and country.
we er show the country or create a new attribute then it'll be more attractive for the user

### CODE REVIEW

**Comment by armudgal** [comment]
Hi @soumyadip007 , thank you for this PR, please make the corresponding issue and discuss it before working on any PR. This helps us discuss whether the proposed feature is worth working on or not and helps us save the time for the contributor. Thanks

**Comment by soumyadip007** [comment]
Okay @armudgal.....so in which field I should start work?

**Comment by soumyadip007** [comment]
I'm really excited to contribute in your org.

File: app/views/users/logix/profile.html.erb [hunk 1]
```diff
@@ -8,9 +8,11 @@
                 <div class="row">
                    <div class="col-lg-8">
                     <p> <strong>Name: </strong><%=@profile.name%></p>
-                    <p> <strong>Member since: </strong><%=((Time.now.to_i - @profile.created_at.to_i)/1.day)%> days</p>
+ <p> <strong>Name: </strong><%=@profile.name%></p>
+					
+				  <p> <strong>Member since: </strong><%=((Time.now.to_i - @profile.created_at.to_i)/1.day)%> days</p>
                     <!--<p> <strong>Educational Institution: </strong> IIIT-Bangalore </p>-->
-                    <!--<p> <strong>Country: </strong> India </p>-->
+                   <p> <strong>Country: </strong> <%=@profile.country%> </p>
```
Comment (satu0king) [inline app/views/users/logix/profile.html.erb]: This would work only if country was already an attribute in the model.

**Comment by satu0king** [comment]
Hello, Why have you add your proposal as a file in this PR?

**Pull request closed without merging**